contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -1,63 +1,284 @@
 var Backbone = require("backbone");
 var _= require("underscore");
 
-window.ChannelEditDataModel = Backbone.Model.extend({
-	urlRoot: '/channel_edit',
-	initialize: function() {
-		//_.bindAll(this);
-	}
-});
-
-/* TODO: CHANGE TO ChannelEditDataModel.EXTEND */
-var TopicNodeModel = Backbone.Model.extend({
+var NodeModel = Backbone.Model.extend({
 	defaults: {
-        color1:"",
-		color2:"",
-		color3:"",
 		title:"Untitled",
-		description:"This is the description generated by default"
-    }
+		description:"No description",
+		parent: null,
+		children:[],
+    },
+    urlRoot: function() {
+		return window.Urls["node-list"]();
+	},
+	toJSON: function() {
+	  var json = Backbone.Model.prototype.toJSON.apply(this, arguments);
+	  json.cid = this.cid;
+	  return json;
+	},
+	parse: function(response) {
```

Comment:
If we are doing a GET on a NodeModel, we should only ever be receiving one object back - why is this ```parse``` necessary?